### DISCUSSION TITLE
Cannot connect to wallet on mobile both Android and IOS

### DISCUSSION THREAD

**Discussion by xinkin** [rainbow-me/rainbowkit#1882] [Q&A] [answered]
### Problem:
On Android and IOS clicking on metamask for wallet connect does not work. 

### Issues:
- Wallet Connect Modal opens up properly but the metamask and rainbow wallet options do not work.
- neither opens metamask nor does it redirect.
- Metamask Icon is like a dummy button, it works properly on the web but not on mobile.

https://github.com/rainbow-me/rainbowkit/assets/91452674/61432229-0a4c-4bf5-9edf-a60b5172536d

**Comment by 0xmkh** [comment]
@xinkin Can you share a small reproducible example ? 

Also can i ask you a follow up questions to make sure i get a clear understanding of what is happening ?
1. Are you using your own `projectId` ?
2. Do you have any of those wallets on android ?

**Reply by xinkin** [reply]
to reproduce it I just followed Rainbow Kit Docs to set up a wallet connection on my website,
1. No, I haven't been using my own `projectId` Thank you for pointing that out I will try it.
2. Yes the Wallets are installed on android

**Reply by xinkin** [reply, marked as answer]
The issue has been resolved, it was because of the `projectId` on being new
Thank you so much @magiziz

**Reply by 0xmkh** [reply]
Glad it worked 🙏

**Comment by PPnaruto** [comment]
Hey @xinkin, I am facing the same issue , Can you guide me, how did you resolve the issue? 
I have added our projected but still issue persisted the same. Is there any additional steps to follow ?

**Reply by ethanpx** [reply]
> Hey @xinkin, I am facing the same issue , Can you guide me, how did you resolve the issue? I have added our projected but still issue persisted the same. Is there any additional steps to follow ?

Maybe you're missing the projectId. If you don't have one yet, you can register a new one in [walletconnect cloud](https://www.rainbowkit.com/docs/installation#:~:text=Note%3A%20Every%20dApp%20that%20relies%20on%20WalletConnect%20now%20needs%20to%20obtain%20a%20projectId%20from%20WalletConnect%20Cloud.%20This%20is%20absolutely%20free%20and%20only%20takes%20a%20few%20minutes.)

**Reply by harsh7800** [reply]
`const connectors = connectorsForWallets(
  [
    {
      groupName: "Recommended",
      wallets: [
        metaMaskWallet,
        walletConnectWallet,
        phantomWallet,
        coinbaseWallet,
        rainbowWallet,
        trustWallet,
      ],
    },
  ],
  { appName: "Iceblox", projectId }
);`


`export const config = getDefaultConfig({
  autoConnect: true,
  connectors: connectors,
  ssr: true,
  appName: "Iceblox",
  projectId: projectId,
  chains: [mainnet, polygon],
  transports: {
    [mainnet.id]: http(),
    [polygon.id]: http(),
    [polygonAmoy.id]: http(),
  },

  // ssr: true,
  // storage: createStorage({
  //   storage: cookieStorage,
  // }),
})`

i have my projectId as well but still i dont see it work on any mobile browser or wallets but works fine in the web, any help will be appreciated stuck on this for more than 3 days now
